app/javascript/active_admin/lib/modal-dialog.js

```diff
@@ -21,13 +21,13 @@ function ModalDialog(message, inputs, callback){
           const result = [];
 
           opts.forEach(v => {
-            const $elem = $(`<${elem}/>`);
+            const $elem = $(`<${elem}></${elem}>`);
```

Comment:
I was trying to figure out what tag goes here. It only can be `<option>`, right? I think the code would be more readable if it was hardcoded, but in any case, it's unrelated to this PR :)

Reply:
That's right. Thanks!